src/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -41,17 +41,9 @@ See the LICENSE file in the project root for more information.
       DependsOnTargets="$(LinkNativeDependsOn)">
 
     <ItemGroup>
-      <NativeLibrary Condition="$(NativeCodeGen) == ''" Include="$(IlcPath)/sdk/libbootstrapper.a" />
```

Comment:
Also, I meant whether we can share all/most the duplicated parts (e.g. the default LinkerArgs, ...) ...